R/RocksDB/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "RocksDB"
+version = v"8.9.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/facebook/rocksdb.git", "49ce8a1064dd1ad89117899839bf136365e49e79")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd rocksdb
+mkdir build && cd build
+
+# build flags for arm
+if [[ "${target}" = arm* ]] || [[ "${target}" == aarch* ]]; then
+    export ARMCRC_SOURCE=1
+    export CXXFLAGS="-march=armv8-a+crc+crypto"
+    export CFLAGS="-march=armv8-a+crc+crypto"
```

Comment:
Why are you forcing these flags?

Also, the `arm*` platforms are actually armv7 and armv6, so you shouldn't try to use an armv8 architecture there.